src/rust/engine/process_execution/src/local.rs

```diff
@@ -117,6 +117,7 @@ impl StreamedHermeticCommand {
   fn new<S: AsRef<OsStr>>(program: S) -> StreamedHermeticCommand {
     let mut inner = Command::new(program);
     inner
+      .kill_on_drop(true)
```

Comment:
Possibly add a comment with some of the PR description included.

Reply:
Good call. I did not apply "PR-description/documentation driven development" here.

Reply:
Huh, I've never heard this phrase. Is this a thing? Like TDD?

Reply:
"documentation driven development" is sortof a thing? google shows https://www.heavybit.com/library/video/how-great-documentation-drives-developer-adoption/

I'm misusing it a bit here, but I have generally thought that simpler systems have simpler documentation. If this did exactly what it looks like it would do, I wouldn't need (as much) documentation.